src/carts/src/main/java/mushop/carts/CartRepositoryDatabaseImpl.java

```diff
@@ -141,5 +146,18 @@ public void save(Cart cart) {
             throw new RuntimeException(e);
         }
     }
+
+    @Override
+    public boolean healthCheck(){
+        try (Connection con = pool.getConnection()) {
+            OracleDatabase db = SODA.getDatabase(con);
+            OracleCollection col = db.openCollection(collectionName);
+            String name = col.admin().getName();
+            return name == null ? false : true;
```

Comment:
you could probably just do `return name != null;`

Reply:
fixed